addons/web/static/src/webclient/navbar/navbar.js

```diff
@@ -52,23 +55,24 @@ export class NavBar extends Component {
         this.actionService = useService("action");
         this.menuService = useService("menu");
         this.appSubMenus = useRef("appSubMenus");
-        const debouncedAdapt = debounce(this.adapt.bind(this), 250);
+        const debouncedAdapt = debounce(whenAlive(this, this.adapt.bind(this)), 250);
```

Comment:
Maybe use throttleForAnimation instead of debounce ?
Also, call cancel() on will unmount ?